--- a/src/modules/simulation/gz_plugins/moving_platform_controller/MovingPlatformController.cpp
+++ b/src/modules/simulation/gz_plugins/moving_platform_controller/MovingPlatformController.cpp
@@ -240,7 +240,7 @@

 		if (accel_xy > max_accel) {
 			const float scaling = max_accel / accel_xy;
-			_force += feedback_force * gz::math::Vector3d(scaling, scaling, 1.);
+			_force += feedback_force * gz::math::Vector3d(static_cast<double>(scaling), static_cast<double>(scaling), 1.0);

 		} else {
 			_force += feedback_force;
