--- a/src/modules/simulation/gz_bridge/GZMixingInterfaceWheel.cpp
+++ b/src/modules/simulation/gz_bridge/GZMixingInterfaceWheel.cpp
@@ -76,7 +76,7 @@

 	if (active_output_count > 0) {
 		gz::msgs::Actuators wheel_velocity_message;
-		wheel_velocity_message.mutable_velocity()->Resize(active_output_count, 0);
+		wheel_velocity_message.mutable_velocity()->resize(active_output_count, 0);

 		for (unsigned i = 0; i < active_output_count; i++) {
 			// Offsetting the output allows for negative values despite unsigned integer to reverse the wheels
